spec/features/merchants/items/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "Merchant Items Show Page" do
+  describe 'when I visit this page' do
+    it "has the Item listed with all of its attributes" do
+      merchant = Merchant.create!(name: "Paul the Merchant")
+      item1 = merchant.items.create!(name: "Paul's Item",
```

Comment:
Great formatting, please call me out of you ever see me not doing this in mind because I need to build that habit!